.github/CODEOWNERS

```diff
@@ -170,7 +170,8 @@ packages/react-components/theme-designer @microsoft/cxe-coastal
 packages/react-components/global-context @microsoft/teams-prg
 packages/react-components/babel-preset-global-context @microsoft/teams-prg
 packages/react-components/react-table @microsoft/teams-prg
-## <%= NX-CODEOWNER-PLACEHOLDER %>
+#packages/react-components/react-progress @microsoft/cxe-red
```

Comment:
Should this be commented out?

Reply:
And should your GH handle also be an owner of this package?

Reply:
Overall the main requirement is that a team like cxe-read is the first owner and primary owner, but yeah in cxe-red and cxe-coastly we have been putting individuals as the second name so we also know who's mainly on point for the component